lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -20,10 +20,26 @@ export class FetchClient implements INetworkModule {
      */
     async sendGetRequestAsync<T>(url: string, options?: NetworkRequestOptions): Promise<NetworkResponse<T>> {
         let response;
+        let controller: AbortController | undefined;
+        let timeoutId: number | undefined;
+
+        /*
+         * Check if a timeout value is provided via the options and create 
+         * a timeout based on the value provided to abort the request
+         */
+        if (options?.timeout) {
+            controller = new AbortController();
+            timeoutId = window.setTimeout(() => controller?.abort(), options.timeout);
```

Comment:
If we want to add timeout logic here (which we should), it should be in both GET and POST, and for both the fetch client and xhr client. Given that, it might be better to have this put into a separate PR, with this PR just containing the caching logic.

Reply:
Got it, I have removed that logic for now.